src/themes/shared/borders.ts

```diff
@@ -1,5 +1,7 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
if you update eslint-pkg in your PR, you don't need that anymore

Reply:
Wuiii. Thank you ! 🥦
I could removed that in the update PR but now it's too late. I will remove this line and the rest in this PR. 😅